services/web/serve-static.js

```diff
@@ -1,9 +1,14 @@
-const fs = require('fs');
-const url = require('url');
 const Koa = require('koa');
-const compress = require('koa-compress');
-const koaStatic = require('koa-static');
-const config = require('@kaareal/config');
+const koaMount = require('koa-mount');
+const koaLogger = require('koa-logger');
+const basicAuth = require('koa-basic-auth');
```

Comment:
note to myself to check out HTTP_BASIC_AUTH_USER and HTTP_BASIC_AUTH_PASS isnt exposed to web. As they are in AD

Reply:
It's intended to be something you can set up via the k8s config if needed ... I'll add blank keys to `env.conf` which I think is good practice yes?

Reply:
Just added defaults in `env.conf` to the new PR. I fixed this for AD a while back so should be good now.